src/components/DialogCloseButton.tsx

```diff
@@ -1,18 +1,17 @@
-import { IconButton } from '@material-ui/core';
```

Comment:
Same here, if we are using the new DialogBox component everywhere, we shouldn't need this CloseButton anymore

Reply:
This and CancelButton and TextField are still being used in terraware-web.
Perhaps something I can use your help cleaning up.
For now I will keep these in the PR, can't do it all in one step.

Reply:
Sure!

Reply:
Thanks 🙏🏻 I’ll go ahead and remove them from this PR. You are correct we don’t need to copy them here.